wikihow2zim/entrypoint.py

```diff
@@ -0,0 +1,66 @@
+# -*- coding: utf-8 -*-
+
+import sys
+import argparse
+
+from .constants import NAME, setDebug, getLogger
+
+def main():
+    parser = argparse.ArgumentParser(
+        prog=NAME,
+        description="Scraper to create ZIM files from Kolibri channels",
+    )
+
+    parser.add_argument(
+        "--language",
+        choices=['en', 'de', 'fr'],
```

Comment:
You should define the list of actual domains we'll support in constants and use it here in the choices and help message.